### PR TITLE
fix: show error to user when online fails to download clipboard on pasting

### PR DESCRIPTION
**Pull request by Rash419**
follow up of e9a0eeb78dcc

Change-Id: I7c3cf0cbb0b0b945b6144d9b2b6a2d8cf02a0a2c

* Target version: master

### CODE REVIEW

**Comment by Rash419** [comment]
Don't merge until I say so I am building docker to test the functionality in cluster environment

**Review by eszkadev — APPROVED** [review]

**Comment by Rash419** [comment]
Okay to merge now, it works as expected:
image

File: browser/src/map/Clipboard.js [hunk 1]
```diff
@@ -334,9 +334,20 @@ L.Clipboard = L.Class.extend({
 				);
 			}.bind(this),
 			function(progress) { return progress/2; },
-			function() {
+			function(response) {
 				window.app.console.log('failed to download clipboard using fallback html');
 
+				if (response) {
+					const reader = new FileReader();
```
Comment (eszkadev) [inline browser/src/map/Clipboard.js:341]: this code is duplicated, please create shared function somewhere

Comment (eszkadev) [reply browser/src/map/Clipboard.js:341]: eg. in docstatefunctions.js then it is accessible using app.nameOfFunction

**Review by eszkadev — CHANGES_REQUESTED** [review]

**Review by eszkadev — APPROVED** [review]

**Pull request merged**